Keas.Mvc/Startup.cs

```diff
@@ -208,51 +272,76 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseExceptionHandler("/Error/Index");
             }
 
+
             app.UseStatusCodePages("text/plain", "Status code page, status code: {0}");
 
-            app.UseStaticFiles();
+            if (env.IsDevelopment())
+            {
+                // dist folder will be served by webpack-dev-server
+                app.UseWhen(context => !context.Request.Path.StartsWithSegments("/dist"),

```

Comment:
Why is this needed? I've never seen it in any of our other apps.

Reply:
If the static file middleware picks up the vendor.js and other files, it doesn't give the webpack-dev-server a jance to serve them and inject the HMR code. Took me forever to figure it out, because I thought it was something to do with having just added SwaggerGen.